content/04.methods.md

```diff
@@ -174,25 +174,27 @@ In contrast, if a merged `SingleCellExperiment` object contained HTO data due to
 #### Data preparation
 
 We identified solid tumor samples with both bulk and single-cell (or single-nuclei) RNA-seq data in the ScPCA Portal for analysis, with multiplexed samples excluded (N=105).
-We further identified low-quality samples based on visual inspection of quality control reports and removed (N=8), leaving a total of 97 samples across five ScPCA projects for analysis.
+We removed low-quality samples based on visual inspection of quality control reports (N=8), leaving a total of 97 samples across five ScPCA projects for analysis.
 
-For each project, we transformed and normalized with respect to library size bulk counts matrices for all samples using `DESeq2::rlog()` [@doi:10.1186/s13059-014-0550-8].
+For each project, we transformed and normalized bulk counts matrices for all samples using `DESeq2::rlog()` [@doi:10.1186/s13059-014-0550-8].
 We obtained pseudobulk counts by summing raw single-cell counts for each sample, and similarly transformed each project's resulting counts matrix with `DESeq2::rlog()`.
 We filtered out genes which were not observed in either the bulk or pseudobulk raw counts matrices before subsequent analysis.
 For each project, we then used the `lme4` [@doi:10.18637/jss.v067.i01] R package to construct a linear model predicting bulk from pseudobulk counts considering a random effect for sample id: `bulk ~ pseudobulk + (1|sample_id)`.
 
 #### Overrepresentation analysis
 
-To identify cell types that might be overrrepresented in one modality compared to the other, we first identified gene sets associated with each cell type of interest, defined as each project's consensus cell types.
-Because the consensus cell types can encompass multiple cell types from the `CellAssign` marker gene reference, we defined the consensus cell type gene sets as the union of all gene sets for the constituent cell types.
+We next asked whether certain cell types might be overrepresented in one modality compared to the other.
+For this, we first identified cell types of interest as all possible consensus cell types for a given project.
```

Comment:
I'm not sure that this is accurate, but I think we want to say something mor explict than "for a given project"
```suggestion
For this, we first identified cell types of interest as all possible consensus cell types that were labeled in each project.
```

Reply:
Yeah, this isn't quite accurate - we did include cell types that were not labeled, but _could_ have been labeled, but I worry now that "could have been labeled" may just raise more questions than it answers. Maybe I'll add a TODO here to really remind us to check phrasing.